src/main/java/com/Incubatormsrv/Incubatormsrv/weather/mapper/weatherMapper.java

```diff
@@ -0,0 +1,11 @@
+package com.Incubatormsrv.Incubatormsrv.weather.mapper;
+
+import com.Incubatormsrv.Incubatormsrv.weather.dto.weatherModelDto;
+import com.Incubatormsrv.Incubatormsrv.weather.model.weatherModel;
+import org.mapstruct.Mapper;
+
+@Mapper
+public interface weatherMapper {
```

Comment:
minor point but the Java convention is to start interface and class names witha capital letter.